conf/test.config

```diff
@@ -20,7 +20,7 @@ params {
     max_time   = '6.h'
 
     // Input data
-    input = 'https://raw.githubusercontent.com/nf-core/test-datasets/cutandrun/samplesheet/test-GSE145187-small.csv'
+    input = 'https://raw.githubusercontent.com/luslab/test-datasets/cutandrun/samplesheet/test-GSE145187-small.csv'
```

Comment:
Can the test data go to nf-core/test-datasets?

Reply:
yes this will happen at the last min after approval